### PR TITLE
Adjusted Gentle Touch Cure spirit sphere cosumption

### PR DESCRIPTION
**Pull request by MrKeiKun**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

Adjusted Gentle Touch Cure, consumes fixed 1 spirit sphere per cast.

**Issues addressed:** 

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

File: db/re/skill_db.conf [hunk 1]
```diff
@@ -26754,18 +26743,7 @@ skill_db: (
...
-			Lv3: 3
-			Lv4: 4
-			Lv5: 5
-			Lv6: 6
-			Lv7: 7
-			Lv8: 8
-			Lv9: 9
-			Lv10: 10
-		}
+		SpiritSphereCost: 2
```
Comment (Kenpachi2k13) [inline db/re/skill_db.conf:26746]: Are you sure this isn't iRO-only?
According to the skill description in kRO and twRO only 1 sphere is required/consumed...

Comment (Kenpachi2k13) [reply db/re/skill_db.conf:26746]: I just tested `Gentle Touch Revitalize` in my twRO zone and it requires/consumes 1 sphere per used skill level.
Well, that zone is EP 14 so maybe the requirement has changed until today. :confused:

Comment (MrKeiKun) [reply db/re/skill_db.conf:26746]: I will do more test on kRO this november...

Comment (MrKeiKun) [reply db/re/skill_db.conf]: so after checking, it does still consumes 1 sphere per level.

**Comment by Kenpachi2k13** [comment]
Sorry for being a pain in the neck but since one can't choose the used level of `SR_GENTLETOUCH_CURE` I couldn't imagine that that the required amount of spheres depends on that, so I blindly trusted the skill description.
Out of curiosity I just tested this in my twRO zone and well.... the skill requires one sphere per used (or rather _learned_) skill level. :unamused:

**Pull request closed without merging**